js/main.js

```diff
@@ -148,66 +153,50 @@ $ul.addEventListener('click', function (event) {
-    $modalBtnWrap.setAttribute('class', 'modal-btn-wrap');
-    $modalWrap.appendChild($modalBtnWrap);
-    const $modalCancelBtn = document.createElement('button');
-    $modalCancelBtn.setAttribute('class', 'cancel btn');
-    $modalCancelBtn.textContent = 'CANCEL';
-    $modalBtnWrap.appendChild($modalCancelBtn);
-    const $modalConfirmBtn = document.createElement('button');
-    $modalConfirmBtn.setAttribute('class', 'confirm btn');
-    $modalConfirmBtn.textContent = 'CONFIRM';
-    $modalBtnWrap.appendChild($modalConfirmBtn);
-
-    $hiddenModal.setAttribute('class', 'visible modal');
-
-    $modalCancelBtn.addEventListener('click', function () {
-      $hiddenModal.setAttribute('class', 'hidden');
-    });
-
-    $modalConfirmBtn.addEventListener('click', function () {
-      const $li = event.target.closest('li');
-      for (let i = 0; i < data.entries.length; i++) {
-        if (data.entries[i].entryId === Number($li.getAttribute('data-entry-id'))) {
-          data.entries.splice(0, 1);
-          data.editing.splice(0, 1);
-        }
-      }
-      $li.remove();
-      toggleNoEntries();
-      $hiddenModal.setAttribute('class', 'hidden');
+  // (above) loops through data entries (each form) in data.js to find the entryId that matches the one from the current selected <li> and adds it to data object's editing property
+
+  const $editTitle = document.querySelector('#title');
+  $editTitle.value = data.editing.title;
+  const $editImg = document.querySelector('#photo-url');
+  $editImg.value = data.editing.imgUrl;
+  const $editImgSrc = document.querySelector('img');
+  $editImgSrc.src = data.editing.imgUrl;
+  const $editNotes = document.querySelector('#notes');
+  $editNotes.value = data.editing.notes;
+  // (above) grabs the title, image url input, image src, and notes input value from the user's past (selected) entry using main.js data object's editing
+
+  // changes the New Entry page heading to Edit Entries
+  $pageName.textContent = 'Edit Entries';
+});
+
+// modal pop up when user clicks delete entry
+const $modal = document.querySelector('.modal');
+
+$delButton.addEventListener('click', function (event) {
+  $modal.classList.remove('hidden');
+});
+// modal cancel delete button and exits modal
+const $modalCancelBtn = document.querySelector('.cancel');
+$modalCancelBtn.addEventListener('click', function () {
+  $modal.classList.add('hidden');
+});
+
+// modal confirm delete button and deletes that form entryId from data storage
+const $modalConfirmBtn = document.querySelector('.confirm');
+
+$modalConfirmBtn.addEventListener('click', function () {
+  $modal.classList.add('hidden');
+  const $li = document.querySelectorAll('li');
+  for (let i = 0; i < data.entries.length; i++) {
+    if (data.entries[i].entryId === Number(data.editing.entryId)) {
+      data.entries.splice([i], 1);
+      $ul.removeChild($li[i]);
+      data.editing = null;
+      data.view = 'entries';
```

Comment:
You shouldn't be setting `data.view` here, you should be setting this inside of `viewSwap`